modules/itk_siteimprove/itk_siteimprove.module

```diff
@@ -0,0 +1,147 @@
+<?php
+
+/**
+ * @file
+ * Add Siteimprove functionality and js.
+ */
+
+/**
+ * Implements hook_menu().
+ */
+function itk_siteimprove_menu() {
+  $items = array();
+
+  $items['admin/config/system/itk-siteimprove'] = array(
+    'title' => 'ITK Siteimprove',
+    'description' => 'Manage ITK Siteimprove settings.',
+    'page callback' => 'drupal_get_form',
+    'page arguments' => array('itk_siteimprove_form'),
+    'access arguments' => array('administer itk siteimprove'),
+  );
+
+  return $items;
+}
+
+/**
+ * Add new permission.
+ *
+ * @return array
+ *   An array defining a permission.
+ */
+function itk_siteimprove_permission() {
+  return array(
+    'administer itk siteimprove' => array(
+      'title' => t('Administer ITK Siteimprove module'),
+      'description' => t('Perform administration tasks related to ITK Siteimprove.'),
+    ),
+  );
+}
+
+/**
+ * Custom form for configuring ITK Siteimprove.
+ *
+ * @param array $form
+ *   Custom configuration form.
+ * @param array $form_state
+ *   Form state of custom form.
+ *
+ * @return mixed
+ *   The complete form.
+ */
+function itk_siteimprove_form(array $form, array &$form_state) {
+  $form['tabs'] = [
```

Comment:
Missing $form = [] to define the array befor using it.

Reply:
Added